panel/io/resources.py

```diff
@@ -122,6 +122,7 @@ def conffilter(value):
 
 extension_dirs['panel'] = str(DIST_DIR)
 
+mimetypes.init()
```

Comment:
To me it looks like the function is idempotent and can be run multiple times and thus its ok for us to trigger it.

Reply:
This is true but I also worry that it might override someone else's overrides, from the docs:

>If files is None the internal data structure is completely rebuilt to its initial default value. This is a stable operation and will produce the same results when called multiple times.